settings/Mailing.setting.php

```diff
@@ -393,4 +393,18 @@
     'description' => ts('The frequency that CiviMail updates its sent mail database.'),
     'help_text' => NULL,
   ],
+  'scheduled_reminder_smarty' => [
+    'group_name' => 'Mailing Preferences',
+    'group' => 'mailing',
+    'name' => 'scheduled_reminder_smarty',
+    'type' => 'Boolean',
+    'html_type' => 'checkbox',
+    'default' => 0,
+    'title' => ts('Use Smarty in scheduled reminders'),
+    'add' => '5.59',
```

Comment:
Now 5.60 - civi moves so fast.

Reply:
also fixed. thanks for your continued reviews @demeritcowboy - appreciated :)